change-beta/@azure-communication-react-cc2ef409-ce98-484f-8b0e-d047cc26af08.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "area": "fix",
+  "workstream": "File Sharing",
+  "comment": "Fixed the issue where RTE cannot send rich text with file attachments",
```

Comment:
could you please use `rich text editor` instead of `RTE`?

Reply:
thanks updated